coralproject/talk
# [CORL-2314]: Update single site ban for admins to use SiteSearch

## What does this PR do?

These changes update the `BanUserModal` to use the `SiteSearch` component. This addresses the issue where if there were many sites, the selector would take awhile to load all the sites and be able to scroll to ban. When a site is selected to be banned on in the modal, a checkbox shows for the selected site. If this checkbox is unchecked, the site is removed from the list of sites to ban across.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

Go to `Community` in the admin. On a user's status, click `Ban`. Select to ban across `Specific sites`. Search by site name to find sites to ban. Only sites where a user has moderation scopes should be shown as options. Select a site from search results and see that it's added as a checkbox to be banned. Test out adding/removing additional sites to ban. Click `Ban User` and see that the ban is applied as desired.
 
## How do we deploy this PR?
